roles/edpm_nova/tasks/configure.yml

```diff
@@ -118,3 +119,24 @@
     mode: '0600'
     owner: nova
     group: nova
+
+- name: Get fact for tls_certs_enabled
+  ansible.builtin.set_fact:
+    tls_certs_enabled: "{{ lookup('env', 'tls_certs_enabled') | default(false, true) }}"
+
+- name: Copy TLS files to the compute node
+  tags:
+    - configure
+    - nova
+  become: true
+  loop:
+    - {"src": "{{ edpm_nova_certs_src }}/{{ inventory_hostname }}-tls.crt", "dest": "{{ edpm_nova_certs_dest }}/tls.crt"}
+    - {"src": "{{ edpm_nova_certs_src }}/{{ inventory_hostname }}-tls.key", "dest": "{{ edpm_nova_certs_dest }}/tls.key"}
+    - {"src": "{{ edpm_nova_cacerts_src }}/TLSCABundleFile", "dest": "{{ edpm_nova_cacerts_dest }}/TLSCABundleFile"}
+  ansible.builtin.copy:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    mode: '0600'
+    owner: nova
+    group: nova
+  when: tls_certs_enabled|bool
```

Comment:
this should be edpm_nova_tls_certs_enabled

Reply:
done